docs/howto_config.md

```diff
@@ -10,19 +10,21 @@ The MicroShift configuration file must be located at `~/.microshift/config.yaml`
 The format of the `config.yaml` configuration file is as follows.
 
 ```yaml
-subjectAltNames:
-  - ""
-nodeName: ""
-nodeIP: ""
-url: ""
+cluster:
+  name: ""
+  baseDomain: ""
 network:
   clusterNetwork:
     - cidr: ""
   serviceNetwork:
     - ""
   serviceNodePortRange: ""
-dns:
-  baseDomain: ""
+node:
+  hostnameOverride: ""
+  nodeIP: ""
```

Comment:
This field is (currently) used for many things that are unrelated to Node/kubelet. Unless that's going to change, I think putting it into a "node" config group is misleading.

Reply:
I can see your point, though I wouldn't call those "many things" unrelated:
We currently use nodeName for a) TLS certs for services on the node, b) in etcd config making the node member, c) as name for the node in kubelet, d) in mDNS to announce the node. In all cases but kubelet, that node name will be a domain name that resolves to the same IP address. In the case of kubelet, the node name _defaults_ to the host name, thus also pointing to the same IP address. It _may_ be overridden (with `--override-hostname`) to something else, of course, but would still need to be resolvable.

Is there another grouping that'd make more sense to you? Should I revert to "nodeName" (although that also suggests "node") or use `hostnameOverride` (although we use it for things unrelated to kubelet)?

Reply:
Maybe. @pacevedom WDYT?